docs/unit-converter-requirements.md

```diff
@@ -0,0 +1,17 @@
+# Make a unit converter
+
+## Interface
+Get creative.
```

Comment:
I agree. At first, I left out the interface section completely. Then I decided later to put it back in just for consistency's sake since the others (not all, but most of my ideas) have an interface, details, and example section. "Get creative" was just another way of me saying "Design it however you like." Later on, when I added the Google example, I didn't feel so bad about the flimsy "get creative" interface.